README.md

```diff
@@ -106,16 +215,81 @@ erDiagram
 
 # Important flagging details
 
+## Sales validation pipeline run modes
+
+The Sales Validation Pipeline (hereafter referred to as “the pipeline”) can be executed in three distinct run modes, depending on the state of the sales data and the specific requirements for flagging:
+
+1. **Initial Run:** This mode is triggered when no sales have been flagged. It's the first step in the pipeline to instantiate tables and flag sales.
+2. **Glue Job:** This mode applies when there are already flagged sales in the system. It's an automated scheduled job that flags new, unflagged sales.
+3. **Manual Update:** This mode is used when sales need to be re-flagged, either due to errors or methodology updates. This allows for the selective re-flagging of sales.
+
+```mermaid
+
+graph TD
+    subgraph "Manual Update Mode"
+        A3{{"Sales must be re-flagged"}}
+        B3[Specify subset in yaml]
+        C3[Run manual_update.py]
+        D3[Increment version if sale already flagged]
+        E3[Assign Version = 1 if sale unflagged]
+        F3[Update flags in default.vw_pin_sale]
+        G3[Save results to S3 with new run ID]
+
+        A3 -->|Manual selection| B3
+        B3 -->|Run update| C3
+        C3 -->|Version check| D3
+        D3 -->|Update process| F3
+        C3 -->|New flag| E3
+        E3 -->|Update process| F3
+        F3 -->|Persist results| G3
+    end
+
+    subgraph "Glue Job Mode"
+        A2[Schedule triggers glue job]
+        B2{{"Some sales are already flagged"}}
+        C2[Ingest data for unflagged sales]
+        D2[Run flagging model within glue job]
+        E2[Write sales data to sale.flag]
+        F2[Join flags to<br>default.vw_pin_sale]
+        G2[Save results to S3 with unique run ID]
+
+        A2 -->|Trigger| B2
+        B2 -->|Process new sales| C2
+        C2 -->|Run model| D2
+        D2 -->|Output flags| E2
+        E2 -->|Join data| F2
+        F2 -->|Persist results| G2
+    end
+
+    subgraph "Initial Run Mode"
+        A1{{"No sales are flagged"}}
+        B1[Run initial_flagging.py]
+        C1[Flag sales as outliers or non-outliers<br>with Version = 1]
+        D1[Join flags to<br>default.vw_pin_sale]
+        E1[Save results to S3 with unique run ID]
+
+        A1 -->|Initial setup| B1
+        B1 -->|Flag sales| C1
+        C1 -->|Store flags| D1
+        D1 -->|Persist results| E1
+    end
+
+    style A1 fill:#f9f,stroke:#333,stroke-width:2px
+    style B2 fill:#bbf,stroke:#333,stroke-width:2px
+    style A3 fill:#fbf,stroke:#333,stroke-width:2px
```

Comment:
**issue:** These styles aren't readable (text is much too light compared to the fill color). Also, why have different colors for each subgraph?

Reply:
I standardized the colors for the subgraphs, but I'm not sure what you mean about the text being light. Seems like the text is all black upon rendering? 

Do you mean that the fill colors are too dark for the text?